test/tools/paths.d

```diff
@@ -32,7 +32,7 @@ else
 
 enum projectRootDir = __FILE_FULL_PATH__.dirName.buildNormalizedPath("..", "..");
 enum generatedDir = projectRootDir.buildPath("generated");
-enum resultsDir = testPath("test_results");
+shared string resultsDir = testPath("test_results");
```

Comment:
Oh so I was thinking of adding a shared static constructor here, s.t. resultsDir in run.d can be overridden by environment variables too?

Reply:
> s.t. resultsDir in run.d can be overridden by environment variables too?

That's what is currently happening, but during program startup. Maybe it would be better to follow the example of dmdPath et al and make it a function.

Reply:
> Maybe it would be better to follow the example of dmdPath et al and make it a function.

Now amended the function version.